### PR TITLE
Test giveSorted

### LINKED ISSUE #193: Tests for dispute

### PR DESCRIPTION
**Pull request by Yashk767**
# Description

Added tests for giveSorted function present in dispute.go

Fixes #193

### CODE REVIEW

File: cmd/dispute_test.go [hunk 1]
```diff
@@ -405,3 +405,60 @@ func TestHandleDispute(t *testing.T) {
...
+			args: args{
+				sortedStakers: []uint32{2, 1, 3, 5},
+				giveSortedErr: errors.New("giveSorted error"),
+			},
+		},
+		{
+			name: "Test 3: When sortedStakers is nil",
+			args: args{
+				sortedStakers: nil,
+			},
```
Comment (ashish10677) [inline cmd/dispute_test.go:444]: No check for the time if the error is "gas limit reached"?

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: cmd/dispute_test.go (hunk 1 above)
Comment (Yashk767) [reply cmd/dispute_test.go:444]: It was not added as It gets stuck in the recursion loop once it enters.

Comment (ashish10677) [reply cmd/dispute_test.go:444]: It should not get stuck if the code is correct imo. Can you put a debugger and check why exactly it is getting stuck. If it is getting stuck, we might need to revisit the code.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: cmd/dispute_test.go (hunk 1 above)
Comment (Yashk767) [reply cmd/dispute_test.go:444]: Added the test case for recursive call now.

File: cmd/dispute_test.go [hunk 2]
```diff
@@ -405,3 +405,70 @@ func TestHandleDispute(t *testing.T) {
...
+		},
+		{
+			name: "Test 4: When error is gas limit reached",
+			args: args{
+				sortedStakers: []uint32{2, 1, 3, 5},
```
Comment (ashish10677) [inline cmd/dispute_test.go:449]: Can you increase the number of sortedStakers for this test. Anywhere around 15-20 will be good. You can keep this test and add one more test with more number of stakers.

**Review by ashish10677 — APPROVED** [review]
LGTM

**Pull request merged**